packages/block-library/src/image/view.js

```diff
@@ -568,6 +568,9 @@ function setStyles( context, ref ) {
 			--wp--lightbox-image-width: ${ lightboxImgWidth }px;
 			--wp--lightbox-image-height: ${ lightboxImgHeight }px;
 			--wp--lightbox-scale: ${ containerScale };
+			--wp--lightbox-scrollbar-width: ${
+				window.innerWidth - document.body.clientWidth
```

Comment:
This may be rare, but if the `body` tag itself has an explicit width, the animation seems to break. This is probably because the `document.body.clientWidth` value used to calculate the scrollbar width changes.

I think `window.innerWidth - document.documentElement.clientWidth` is better.

![body](https://github.com/WordPress/gutenberg/assets/54422211/e7403a2d-a960-46e2-8d03-c1ff0755cba7)

https://github.com/WordPress/gutenberg/assets/54422211/15d2a6d5-1a90-4466-b3b6-5dbbbbcd54fd

Reply:
@t-hamano 
Replaced to documentElement. Thanks!!